<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Actions Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="actions_TV">
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="add_PB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_PB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modify_PB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modify ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="swap_PB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Swap</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="multipleActionsBehaviour_L">
       <property name="text">
        <string>Multiple actions behaviour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="multipleActionsBehaviour_CB">
       <item>
        <property name="text">
         <string comment="globalkeys/config/multipleActions">First</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="globalkeys/config/multipleActions">Last</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="globalkeys/config/multipleActions">None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="globalkeys/config/multipleActions">All</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="default_PB">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_PB">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>actions_TV</tabstop>
  <tabstop>add_PB</tabstop>
  <tabstop>remove_PB</tabstop>
  <tabstop>modify_PB</tabstop>
  <tabstop>swap_PB</tabstop>
  <tabstop>close_PB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
